paper/paper.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'pyOBIS: easy access to taxonomic occurrence records harvested from thousands of datasets'
+tags:
+  - Python
+  - oceanography
+  - marine data
+authors:
+  - name: Scott Chamberlain
+    equal-contrib: true
+  - name: Ayush Anand
+    equal-contrib: true
+    affiliation: 1
+  - name: Tylar Murray
+    corresponding: true
+    affiliation: 2
+  - name: Filipe Fernandes
+    corresponding: true
+  - name: Mathew Biddle
+    corresponding: true
+    affiliation: 3
+affiliations:
+ - name: National Institute of Technology Durgapur, India
+   index: 1
+ - name: IMaRS University of South Florida, US
+   index: 2
+ - name: National Oceanic and Atmospheric Administration, National Ocean Service, Integrated Ocean Observing System, US
+   index: 3
+date: 9 May 2023
+bibliography: paper.bib
+---
+
+# Summary
+The pyOBIS python package provides easy access to taxonomic occurrence records harvested from thousands of datasets. The package uses the API from the Ocean Biodiversity Information System (OBIS), a global open-access data and information clearinghouse on marine data for biodiversity for science, conservation, and sustainable development. Included in the pyOBIS package are built-in functions for accessing data on occurrences, taxon, nodes, checklists and datasets. The package provides easy export of data to Pandas DataFrame to help researchers focus more on analysis rather than data mining, and several included Jupyter notebooks demonstrate example analyses that can be used as a starting point for addressing research questions related to global and local distributions of species across space and time. Coupled together with other libraries like pyDwcViz, it forms an ecosystem of analysing Darwin Core Data with super ease through built-in functions.
+
+# Statement of need
+
+## Main Features
+Here are just a few of things pyOBIS can do:
+* Easy handling of OBIS data, easy fetching without handling the raw API response directly.
+* Built-in functions for occurrence, taxon, node, checklist and dataset endpoints of OBIS API.
```

Comment:
`Built-in (filtering or analysis) functions`? Not sure if we should be more descriptive here. Mostly I'd like to reflect that pyOBIS is not just a data fetcher tool.

Reply:
I think yes, we could get more verbose and to ease reading efforts, add headings to bullets.